src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPage.java

```diff
@@ -0,0 +1,415 @@
+  private WebElement privateMassageButton;
+  @FindBy(css = "#UserStatsMenu")
+  private WebElement userStatsMenu;
+  @FindBy(css = "li.private-allow")
+  private WebElement allowPrivateMassageButton;
+  @FindBy(css = "li.kick")
+  private WebElement kickUserButton;
+  @FindBy(css = "li.private-block")
+  private WebElement blockPrivateMassageButton;
+  @FindBy(css = "h1.public.wordmark.selected")
+  private WebElement mainChatSelection;
+  @FindBy(css = "ul.PrivateChatList span.splotch")
+  private WebElement privateMessageNotification;
+  @FindBy(css = ".continued.inline-alert")
+  private WebElement chatInlineAlertContinued;
+  @FindBy(css = "#UserStatsMenu li.ban")
+  private WebElement banUserButton;
+  @FindBy(css = "#ChatBanModal")
+  private WebElement chatBanModal;
+  @FindBy(css = "#ChatBanModal button.primary")
+  private WebElement chatBanModalButton;
+  @FindBy(css = "#UserStatsMenu .regular-actions li")
+  private List<WebElement> userDropDownActionsElements;
+  @FindBy(css = "#UserStatsMenu .admin-actions li")
+  private List<WebElement> adminDropDownActionsElements;
+  @FindBy(css = "#Rail h1.private")
+  private WebElement privateMessagesHeader;
+  @FindBy(css = "#Rail h1.wordmark")
+  private WebElement chatWordmarkImage;
+  @FindBy(css = "#WikiChatList li")
+  private WebElement chatLoadedIndicator;
+  @FindBy(css = "#ChatHeader h1.private")
+  private WebElement privateChatHeader;
+  @FindBy(css = "div.actions .admin-actions .give-chat-mod")
+  private WebElement addUserModStatusButton;
+  @FindBy(css = "div#UserStatsMenu.UserStatsMenu")
+  private WebElement userOptions;
+  @FindBy(css = "ul.admin-actions")
+  private WebElement userModOptions;
+  @FindBy(css = "li.message-wall")
+  private WebElement userMessageWallButton;
+  @FindBy(css = "div.speech-bubble-message")
+  private WebElement userPageMessageWallTab;
+  @FindBy(css = "li.contribs")
+  private WebElement userContributionsButton;
+  @FindBy(css = "table.mw-contributions-table")
+  private WebElement userPageContributionsTab;
+  @FindBy(css = "form#Write.Write.blocked")
+  private WebElement userBlockedMessageField;
+  @FindBy(css = "div.header-column.header-title")
+  private WebElement permissionsErrorTitle;
+  @FindBy(xpath = "//h1")
+  private WebElement userNameTitle;
+
+  private static final String USER_UNBAN_LINK = "//a[@data-type='ban-undo' and @data-user='%s']";
+  private static final String USER_UNBAN_CONFIRM_MESSAGE =
+          "//div[@class='Chat']//li[contains(text(), 'has ended the Chat ban for %s')]";
+  private static final String USER_SELECTOR = "#user-%s";
+  private static final String PRIVATE_MESSAGE_USER_SELECTOR = "#priv-user-%s";
+  private static final String PATH_MESSAGE_ON_CHAT = "//span[@class='message'][contains(text(), '%s')]";
+  private static final String NOTIFICATION_COUNTER =
+          "//span[@class='splotch' and contains(text(), '%s')]";
+
+  private static final int REGULAR_USER_DROPDOWN_ELEMENTS = 3;
```

Comment:
this const doesn't store the dropdown's elements, it stores the count of elements. Let's name this const in a way it says what it is, for instance: `REGULAR_USER_DROPDOWN_ELEMENTS_COUNT`